.github/scripts/binutils/build.sh

```diff
@@ -53,6 +53,10 @@ echo "::group::Build binutils"
     make $BUILD_MAKE_OPTIONS
 echo "::endgroup::"
 
+echo "::group::Execute binutils tests"
+    $ROOT_PATH/.github/scripts/toolchain/execute-binutils-tests.sh
```

Comment:
I would prefer to exclude this step from build.sh
Testing has different purpose and should not block build.

Reply:
I agree it's not ideal but `build.sh` is the only place where `build/binutils` folder is available, because we delete it at the end of `build.sh`. Having said that, why do we delete the build folder at the end of each step (e.g. right after building binutils), isn't the whole Ubuntu image discarded anyways after the job is finished?

Reply:
I guess DELETE_BUILD should be changed to 0 by default

Reply:
> I guess DELETE_BUILD should be changed to 0 by default.

Good point, I'd rather see the binutils build folder deleted after the testing step.

Reply:
Yes, maybe we need to introduce something like clean-build.sh

Reply:
What's the purpose of explicitly deleting the build? It will be discarded with the rest of data in the Ubuntu image when the build job finishes, won't it?

Reply:
This option was added because we were running out of disk space on the GitHub runner. Its semantics is basically when some build directory is no longer needed, delete it so the space is preserved.